packages/menu/src/Menu.spec.js

```diff
@@ -103,4 +108,49 @@ describe('packages/Menu', () => {
       expect(secondItem.rel).toBe('help');
     });
   });
+
+  describe('packages/sub-menu', () => {
```

Comment:
Think it's a little confusing to name it this way, as sub-menu isn't its own package. I think just naming it `SubMenu` is fine. We don't really have hard conventions here, do we?

Reply:
I think we have been doing it for subcomponents, but you're right, it doesn't make sense. I'll fix here and can take a look around the repo and make sure it's cleaned up there too.